### PR TITLE
Fix slow uploads

### PR DESCRIPTION
**Pull request by abhinavkgrd**
## Description

- Renabled cf worker for uploading files through cf worker proxy similar to download 

## Test Plan

- [x] tested locally 
- [x] pending rate limiting test

### CODE REVIEW

File: apps/photos/src/services/userService.ts [hunk 1]
```diff
@@ -493,3 +494,15 @@ export const updateMapEnabledStatus = async (newStatus: boolean) => {
...
+
+export async function getDisableCFUploadProxyFlag() {
+    try {
+        const featureFlags = (
+            await fetch('https://static.ente.io/feature_flags.json')
+        ).json() as GetFeatureFlagResponse;
+        return featureFlags.disableCFUploadProxy;
+    } catch (e) {
+        logError(e, 'failed to get feature flags');
+        return undefined;
```
Comment (ua741) [inline apps/photos/src/services/userService.ts:506]: Why not return a default value here with CF worked as disabled or enabled

File: apps/photos/src/services/userService.ts [hunk 2]
```diff
@@ -493,3 +494,15 @@ export const updateMapEnabledStatus = async (newStatus: boolean) => {
         throw e;
     }
 };
+
+export async function getDisableCFUploadProxyFlag() {
```
Comment (ua741) [inline apps/photos/src/services/userService.ts:498]: Missing return type

File: apps/photos/src/utils/common/apiUtil.ts [hunk 3]
```diff
@@ -45,7 +45,7 @@ export const getPublicCollectionThumbnailURL = (id: number) => {
 };
 
 export const getUploadEndpoint = () => {
-    const endpoint = process.env.NEXT_PUBLIC_ENTE_ENDPOINT;
+    const endpoint = process.env.NEXT_PUBLIC_ENTE_UPLOAD_ENDPOINT;
```
Comment (ua741) [inline apps/photos/src/utils/common/apiUtil.ts:48]: What's this url change? As someone  without much understanding of the code base, I think based on flag we would choose different values

File: apps/photos/src/services/userService.ts (hunk 1 above)
Comment (abhinavkgrd) [reply apps/photos/src/services/userService.ts:506]: ahh yes will update

File: apps/photos/src/utils/common/apiUtil.ts (hunk 3 above)
Comment (abhinavkgrd) [reply apps/photos/src/utils/common/apiUtil.ts:48]: so, I updated the add a new variable -> `NEXT_PUBLIC_ENTE_UPLOAD_ENDPOINT`, earlier it was api was never used and was pointing to the ENTE_ENDPOINT as default value.

**Review by ua741 — APPROVED** [review]

**Pull request merged**